### ISSUE TITLE
Typo in error message for object-curly-newline

### ISSUE THREAD

**Issue by sholladay** [eslint/eslint#6725]
**What version of ESLint are you using?**
2.13.1

**What parser (default, Babel-ESLint, etc.) are you using?**
babel-eslint

**Please show your full configuration:**
```js
{ 'object-curly-newline' : 'error' }
```

**What did you do? Please include the actual source code causing the issue.**
```sh
eslint file.js --env=es6 --rule='object-curly-newline: [2]'
```
```js
// file.js
const foo = {
    one : 1, two : 2
};
```

**What did you expect to happen?**
Receive a grammatically correct error message.
```
Unexpected line break after opening brace
```

**What actually happened? Please include the actual, raw output from ESLint.**
Received a grammatically incorrect error message.
```
Unexpected a line break after this open brace
```
Key words being: "Unexpected a line"

---

I may not have time to create a PR for a while. Filing this so it doesn't get forgotten.

**Comment by platinumazure** [comment]
Not sure if a rule message error counts as a bug or merely a docs change-- my assumption is bug because it involves a code change. Thus labeling bug/evaluating/rule.

I'm :+1: for this change, personally.

**Comment by ilyavolodin** [comment]
It's a bug, and I don't think we need +1s for this. I'll mark it as accepted.

**Comment by mysticatea** [comment]
The article is difficult for me (my language does not have it). :sweat_smile:

**Comment by vitorbal** [comment]
I'll take care of this.

**Comment by nzakas** [comment]
Remember, bugs can be marked as accepted if you can reproduce. Only enhancement requests and new rules need consensus.
